.slashrc

```diff
@@ -122,6 +122,16 @@ class MediaPlugin(slash.plugins.PluginInterface):
       os.remove(tstfile)
     return tstfile
 
+  def _purge_test_artifact(self, filename):
+    result = slash.context.result
+
+    if self.retention != self.RETENTION_ALL:
+      if self.retention == self.RETENTION_FAIL and not result.is_success():
```

Comment:
Ah, `result.is_success()` is not a valid call during test run time... it will always return True until test ends or if test explicitly adds a fail status before this is called.

Thus, this function should only remove the file if `self.retention == self.RETENTION_NONE`

Reply:
thanks, added "if self.retention == self.RETENTION_NONE" check

Reply:
My apologies... on second thought, the original logic does actually make sense.  If it's no trouble can you disregard this change.

Reply:
ok, reverted it